app/models/enterprise.rb

```diff
@@ -317,7 +317,7 @@ def category
   def distributed_taxons
     Spree::Taxon.
       joins(:products).
-      where('spree_products.id IN (?)', Spree::Product.in_distributor(self)).
+      where(spree_products: { id: Spree::Product.select(:id).in_distributor(self) }).
```

Comment:
I guess this would look better? is it not the same?
Spree::Product.in_distributor(self).select(:id)